.github/workflows/premerge.yaml

```diff
@@ -10,6 +10,7 @@ on:
   push:
     branches:
       - 'main'
+      - 'release/**'
```

Comment:
Do you only want this for post-submit testing on the release branch? Is this just for testing currently similar to what we're doing for Linux?

Reply:
I want to do pre-commit and post-commit testing, which is what we are currently doing on the release branch.

Reply:
I would like to have the mac testing ready by the LLVM 20 branch date.  For Windows and Linux we are already covered on the release branch by buildkite, so there is no rush to start using the new GitHub Actions workflow for those platforms.

If the new workflow isn't ready.  I can patch the workflow file in the release branch to drop the:

```
   paths:
      - .github/workflows/premerge.yaml
```
And also disable the linux job.

Reply:
Sounds good. I'd like to have everything ready by then, but can't make any guarantees. We'll probably have to backport some patches to the release branch as once we validate the new premerge workflow, we want to stop running the Buildkite pipeline, but they should be lowrisk (only impacting CI), and I can help with that.